2014-08-27 09:47:44,169 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:47:46,728 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:47:46,744 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:47:48,102 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:47:48,117 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:47:49,008 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:47:49,008 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:48:20,213 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:48:20,213 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 09:48:23,053 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:48:23,068 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:49:46,594 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:49:46,594 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 09:49:49,325 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:49:49,325 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:49:50,839 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:49:50,839 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:53:06,704 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 09:53:06,704 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:53:09,420 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:53:09,435 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:53:10,996 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:53:11,012 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:53:11,871 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:53:11,871 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-27 09:54:56,673 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:54:56,673 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 09:54:59,653 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:54:59,669 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:56:28,053 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 09:56:28,053 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 09:56:29,192 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:56:29,208 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-27 09:56:30,628 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:56:30,644 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 09:56:32,064 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 09:56:32,080 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 10:10:16,824 ERROR [Mina] - New Connect VIN:A0000038C19511EV3
2014-08-27 10:10:16,824 ERROR [Mina] - A0000038C19511EV3 is offline!
2014-08-27 10:10:18,010 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 10:10:18,025 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-27 10:10:19,446 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 10:10:19,462 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-27 10:10:20,882 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-27 10:10:20,898 ERROR [Mina] - A0000038C19511EV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
